RUNNING_DOCKER.RST

```diff
@@ -0,0 +1,369 @@
+
+- `Running Your Docker Containers`_
+.. _Running Your Docker Containers: https://github.com/SFDO-Tooling/MetaCI/blob/feature/docker/CONTRIBUTING.rst#running-your-docker-containers
+
+
+.env File Creation and Variable Declaration
+-------------------------------------------
+
+*Please begin by making a copy of env.example and renaming it .env in your root project directory*
+
+Variables Required Locally
+---------------------------------
+
+DJANGO_SECRET_KEY: 
+    This represents the secret key for the django web application, for local testing, 
+    arbritary strings such as the one given in the env.example will suffice. 
+    Otherwise use your production secret key.
+
+POSTGRES_USER: 
+    Environment variable set in the docker-compose.yml file under the postgres service, 
+    represents database user. This value has already been configured for you unless 
+    you decide to reconfigure it.
+
+POSTGRES_PASSWORD: 
+    Environment variable set in the docker-compose.yml file under the postgres service,
+    represents database password.This database is configured with no password for 
+    development purposes so leave as is unless changing for production purposes.
+
+POSTGRES_DB:
+    Environment variable set in the docker-compose.yml file under the postgres service,
+    represents database. This variable has already been set to the proper 
+    value `metaci` for the user.
+
+GITHUB_USERNAME:     
+    This represents the username of either the tester or service account configured for MetaCI
+
+GITHUB_PASSWORD:      
+    This represents the password or personal access token a user must have to access 
+    their account a personal access token will be used when Multi Factor Authentication is enabled.
+
+If you need to generate a personal access token please visit the following: 
+https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line
+
+SFDX_CLIENT_ID:       
+    This tells sfdx the client id of the connected app to use for connecting to 
+    the Dev Hub to create scratch orgs (so it's only needed for running plans that use a scratch org). 
+    For SFDO staff it's easiest to use an existing connected app, so its best to ask another team member. 
+    External users setting up MetaCI will need to create their own connected app, 
+    which they can do in the Dev Hub org. 
+    You can adapt these instructions https://cumulusci.readthedocs.io/en/latest/tutorial.html#creating-manually 
+    but there is a difference for MetaCI: because it's connecting to the org non-interactively, 
+    the connected app needs to be set up to use the JWT oauth flow. 
+    That means when creating the connected app the user needs to check the "Use Digital Signatures" 
+    box and upload a certificate. If you are a member of SFDO please visit the following
+    quip link https://salesforce.quip.com/sm9bA3R6GwiK, and look under section 5 of SFDX
+    Command Line Tool Setup & Install
```

Comment:
Let's leave out the quip link and just say that SFDO staff should ask Release Engineering.

For non-staff, I'd like to spell out the steps in a bit more detail so they don't need to put different pieces together on their own. But we can follow up and do that in a separate pull request.

Reply:
Ok will remove but just curious why no link? I locked it down so only added users can view.

Reply:
@jofsky I guess I'm just nervous that someone could change the sharing settings. And I'm also anticipating that the recommended steps for staff will change soon once we set up SSO for the Dev Hub org.